plugins/org.komodo.core/src/config/komodo.cnd

```diff
@@ -284,3 +284,66 @@
   - tko:ruleType (string) = 'SNS' mandatory autocreated < 'SNS'
   - tko:matchType (boolean) = 'true' mandatory
 
+/*
+ * A grouping node for searches made on the repository
+ */
+[tko:searches] > nt:unstructured
+ + * (tko:search) copy
+
+/*
+ * A configuration for building a search query on the repository
+ */
+[tko:search] > nt:unstructured
+ - tko:searchDate (string) mandatory
+ - tko:customWhereClause (string)
+ + * (tko:fromType) copy sns
+ + * (tko:whereClause) copy sns
+
+/*
+ * A From clause for a search
+ */
+[tko:fromType] > nt:unstructured
+ - tko:type (string) mandatory
+ - tko:alias (string)
+
+/*
+ * Where clause for a search
+ */
+[tko:whereClause] > nt:unstructured abstract
+ - tko:preClauseOperator (string) < 'AND', 'OR'
```

Comment:
No because the first clause in the list contains no pre-clause operator. So in WHERE NAME = 'blah AND AGE = '6', the NAME Compare clause has no pre-clause operator while the AGE Compare clause has its pre-clause operator set to AND.